source/content/caching-advanced-topics.md

```diff
@@ -52,6 +52,21 @@ For CSS or JavaScript changes, Drupal and WordPress each offer methods to ensure
 
 Dev and Multidev environments do not cache static assets.
 
+Another solution to consider when debugging issues related to stale static assets is to load the file via a custom PHP script that sets a `no-cache` header value, for example: 
+
+```php
+<?php
+
+// Punch through the Pantheon edge cache.
+header("Cache-Control: no-cache, must-revalidate");
```

Comment:
Might want to have the sample be to cache for one day, or maybe two different lines (one with no-cache, one with a custom TTL) and they can uncomment the one they want